pegen/pegen.c

```diff
@@ -373,50 +373,107 @@ dedent_token(Parser *p)
     return expect_token(p, DEDENT);
 }
 
+
```

Comment:
These two functions are unchanged from ast.c right?

Reply:
Yeah, except for some arguments that we don't need now (the compiling structure pointer).